app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
@thesuss  - yes, indeed .... and so I will focus on checking the swedish-to-english translations.

In this case, google translate is wrong.

However, it looks like rails i18n implementation is smarter than GT.  When I run this branch on my machine (who locale == english), the translated flash message is correct:

<img width="420" alt="screen shot 2016-12-22 at 5 50 37 am" src="https://cloud.githubusercontent.com/assets/9968213/21423354/a1e5eeec-c80a-11e6-9ae6-556cda5b0eb3.png">